genestack_client/files_util.py

```diff
@@ -315,21 +315,25 @@ def get_special_folder(self, name):
 
     def share_files(self, accessions, group, destination_folder=None, password=None):
         """
-        Share files.
+        Share files and links them to folder.
 
-        :param accessions: files accessions
-        :type accessions: list
+        :param accessions: accession or collection accessions to be shared
```

Comment:
`collection of accessions`
Or I'd better use more explicit notion: `list/tuple/set`, because dictionary, for example, is a collection too.